docs/optimization.rst

```diff
@@ -0,0 +1,37 @@
+Memory optimization
+===================
+
+Executable code size
+--------------------
+In order to manage the executable code size, the library can be compiled enabling or disabling several profiles.
+To add or remove a profile from the library, edit the ``client.config`` file.
+More information can be found at: :ref:`micro_xrce_dds_client_label`.
+
+Runtime size
+------------
+The client is static memory free.
+This implies that all memory charge depends of how the stack grows during the execution.
+To control the stack, there are some values that can be modified and impact to a large degree:
+
+Stream buffers
+~~~~~~~~~~~~~~
+1. Make sure you defined only the streams that will be used.
+   You can define a maximun `127` best effort streams and `128` reliable streams,
+   but for the majory of purposes, only one streams of either best effort or reliable can be used.
```

Comment:
It should be `only one stream`, should'nt?

Reply:
You are right. Done!